src/pages/Admin/Venue/Rooms/RoomsForm.tsx

```diff
@@ -320,29 +322,20 @@ const RoomInnerForm: React.FC<RoomInnerFormProps> = (props) => {
                   </div>
                   <div className="toggle-room">
                     <div className="input-title">Enabled ?</div>
```

Comment:
```suggestion
                    {/* @debt pass the header into Toggler's 'label' prop instead of being external like this*/}
                    <div className="input-title">Enabled ?</div>
```